.github/workflows/sync-check.yaml

```diff
@@ -0,0 +1,54 @@
+name: Daily Seed Synchronization Check
+on:
+  push:
+    branches: [PM-1764-create-daily-seed-synchronisation-check]
+  workflow_dispatch:
+  schedule:
+      - cron: '0 0 * * 2,5' # At 00:00 on Tuesday, and Friday
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+jobs:
+  parse-mainnet-seed-list:
+    runs-on: minafoundation-xlarge-runners
```

Comment:
question: any reason you chose to use xlarge runner just to do `cat networks/mainnet.txt`? seems a bit excessive

Reply:
It's a good catch @simisimis. I originally put this runner as we had issue with the `minafoundation-default-runners`. I am reverting back!